libraries/botbuilder-azure-blobs/src/blobsTranscriptStore.ts

```diff
@@ -141,8 +141,7 @@ export class BlobsTranscriptStore implements TranscriptStore {
             const fromIdx =
                 startDate != null
                     ? blobItems.findIndex(
-                          (blobItem) =>
-                              blobItem?.metadata?.timestamp && new Date(blobItem.metadata.timestamp) >= startDate
+                          (blobItem) => blobItem?.properties?.createdOn && blobItem?.properties?.createdOn >= startDate
```

Comment:
I logged the `blobItems` array and found that `metadata` is undefined by default, so I opted for this property that does exist. I tested against a real blob storage container.

Reply:
Is `createdOn` already a `Date()` obj?

This change still works with local storage emulation, too?

Reply:
It is, yes. I'll test with the emulator as well. It should as the Typescript type definitions _should_ be correct. Will double check though.

Reply:
@mdrichardson Emulator checks out!